generic/tests/cfg/boot_vm_in_hugepage.cfg

```diff
@@ -10,8 +10,8 @@
     pre_command_noncritical = yes
     pre_command = "echo 3 > /proc/sys/vm/drop_caches"
     variants:
-        - 2M_hugepage:
-            expected_hugepage_size = 2048
+        - @default:
```

Comment:
My concern is that if someone sets the kernel line wrong or the host doesn't support 1G hugepage, as no "expected_hugepage_size" in cfg, the code will not check it, case will run with the default hugepage size and testers may have not notice about it. 
And I prefer 'extra_params += " -mem-path /mnt/kvm_hugepage"'  than the pc-dimm, which tests hotplugged hugepage memory actually.  Thanks.